src/daemon/krillserver.rs

```diff
@@ -576,47 +577,37 @@ impl KrillServer {
         }
         structure.validate_ca_hierarchy(existing_cas)?;
 
-        if !self.config.ta_proxy_enabled() {
-            Err(Error::custom(
-                "Import CAs is only possible when ta_support_enabled = true",
-            ))
-        } else if !self.config.ta_signer_enabled() {
-            Err(Error::custom(
-                "Import CAs is only possible when ta_signer_enabled = true",
-            ))
-        } else {
-            if let Some(publication_server_uris) = structure.publication_server.clone() {
-                info!("Initialising publication server");
-                self.repo_manager.init(publication_server_uris)?;
-            }
-
-            if let Some(import_ta) = structure.ta.clone() {
-                info!("Creating embedded Trust Anchor");
-                let (ta_aia, ta_uris) = import_ta.into_uris();
-                self.ca_manager
-                    .ta_init_fully_embedded(ta_aia, ta_uris, &self.repo_manager, &actor)
-                    .await?;
-            }
+        if let Some(publication_server_uris) = structure.publication_server.clone() {
```

Comment:
Why did you remove these "enabled" checks?

Reply:
apologies for the late response - got side tracked - they are removed because technically we could just import local CAs and not set up a TA. That said, the checks should be done when a TA is imported - so I will add them there now.